test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1

```diff
@@ -237,4 +237,54 @@ try {
             }
         }
     }
+
+    Context 'Test programmatic Control+Break to enter the debugger' {
+        BeforeAll {
+            $debuggerStopHandler = {
+                param($s, $e)
+                $e.ResumeAction = [System.Management.Automation.DebuggerResumeAction]::Stop
+            }
+
+            function InvokeAsyncHelper {
+                param(
+                    [powershell]$PowerShell
+                )
+                $host.Runspace.Debugger.add_DebuggerStop($debuggerStopHandler)
+                try {
+                    $r = $PowerShell.AddScript("@(1,2,3,4,5)${sbStub}").InvokeAsync()
+                    $host.Runspace.Debugger.SetDebuggerStepMode($true)
+                    if ($Wait) {
+                        [System.Threading.Tasks.Task]::WaitAll(@($r))
+                    }
+                    $r
+                } finally {
+                    $host.Runspace.Debugger.remove_DebuggerStop($debuggerStopHandler)
+                }
+            }
+
+            function GetInnerErrorId {
+                param(
+                    [exception]$Exception
+                )
+                while ($null -ne $Exception.InnerException) {
+                    $Exception = $Exception.InnerException
+                    if ($null -ne (Get-Member -InputObject $Exception -Name ErrorRecord)) {
+                        $Exception.ErrorRecord.FullyQualifiedErrorId
+                        break
+                    }
+                }
+            }
+        }
+
+        It 'can invoke a single script asynchronously, but only in a new runspace' {
+            $ps = [powershell]::Create()
+            try {
+                $r = InvokeAsyncHelper -PowerShell $ps -Wait
+                $r.Status | Should -Be ([System.Threading.Tasks.TaskStatus]::RanToCompletion)
```

Comment:
This test was problematic in PowerShell. I got it to the point where it should have worked, but Pester was throwing exceptions because of how it uses some of the same PowerShell functionality, so I discarded this test in favor of the `TestRemoteDebuggerBreak` unit test written in C#, which covers this need but in a reliable, working fashion.